src/data/markdown/docs/20 jslib/01 jslib/04 utils/44 uuidv4().md

```diff
@@ -6,10 +6,16 @@ excerpt: 'uuid v4 function'
 
 Function returns a random uuid v4 in a string form.
 
+### Parameters
+
+| Parameter         | Type    | Description                                                                                                                                                                                                                                                                                                                                    |
+| :---------------- | :------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| secure (optional) | boolean | As a default, the function uses a standard random number generator. By setting the `secure` option to `true`, `uuidv4` will use a cryptographically secure random number generator instead for added security. Note that using the `secure` option will make the function an order of magnitude slower compared to its standard counterpart. |
```

Comment:
```suggestion
| secure (optional) | boolean | By default, `uuidv4()` uses a standard random number generator. If the `secure` option is set to `true`, `uuidv4` uses a cryptographically secure random number generator instead. While this adds security, the `secure` option also makes the function an order of magnitude slower. |
```

Reply:
Besides these edits for economy and precision, should the reference really say "an order of magnitude" slower instead of "considerably slower?" 

I believe "order of magnitude" has a precise mathematical meaning, something about powers of ten and log scales:
https://en.wikipedia.org/wiki/Order_of_magnitude